app/assets/stylesheets/pages/home/_banner.scss

```diff
@@ -14,7 +14,13 @@
 		left: 50%;
 		top: 50%;
 		transform: translate(-50%, -50%);
-    width: 100%;
+		width: 100%;
+
+		@media only screen and (max-width: 611px){
```

Comment:
what's the point of this media query? it's so useless

Reply:
hmm sorry